### PR TITLE
Make timers namespace specific and add fast Always Enabled support 

### PR DESCRIPTION
**Pull request by gtjoseph**

### CODE REVIEW

**Comment by thebigredgeek** [comment]
@TooTallNate thoughts?

**Comment by freewil** [comment]
Name-spacing the timers makes sense to me. In the past, I found the global timer behavior very non-intuitive when I had multiple workers with different namespaces logging.

File: debug.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ var prevColor = 0;
  * Previous log timestamp.
  */
 
-var prevTime;
+var prevTime = {};
```
Comment (thebigredgeek) [inline debug.js]: In this case, the previous time should be held inside of the debug closure.  That way the value is encapsulated by the log instance.  Technically, this line is a leak

**Comment by thebigredgeek** [comment]
@gtjoseph bump?

**Comment by gtjoseph** [comment]
Wasn't paying attention, sorry. :)

I'll get everything fixed up tonight.

**Comment by gtjoseph** [comment]
Messed up.  Splitting the pull request into 2 separate ones.

**Pull request closed without merging**